File: App/Backend/system_obslugi_serwisu/Domain/RepairShops/OpeningHours.cs
namespace system_obslugi_serwisu.Domain.RepairShops;

public class OpeningHours
{
    public TimeInterval? Monday { get; set; }
    public TimeInterval? Tuesday { get; set; }
    public TimeInterval? Wednesday { get; set; }
    public TimeInterval? Thursday { get; set; }
    public TimeInterval? Friday { get; set; }
    public TimeInterval? Saturday { get; set; }
    public TimeInterval? Sunday { get; set; }
}

File: App/Frontend/system_obslugi_serwisu/src/app/Types/OpeningHours.ts
type OpeningInterval = {
    from: string;
    to: string;
} | null;

export type OpeningHours = {
    monday: OpeningInterval;
    tuesday: OpeningInterval;
    wednesday: OpeningInterval;
    thursday: OpeningInterval;
    friday: OpeningInterval;
    saturday: OpeningInterval;
    sunday: OpeningInterval;
}